templates/auto_master.erb

```diff
@@ -0,0 +1,8 @@
+#
+## This file is maintained by puppet. Manual changes will be overwritten.
+#
+
+<% if @mounts -%>
+<% @mounts.each do | mount| -%>
```

Comment:
Is an array ok here or do we need a hash? I'm not familiar enough to know if you need to specify per mount options or if that is even a possibility.